app/celery/letters_pdf_tasks.py

```diff
@@ -474,14 +475,14 @@ def process_virus_scan_error(filename):
     raise error
 
 
-def update_letter_pdf_status(reference, status, billable_units):
+def update_letter_pdf_status(reference, status, billable_units, recipient_address=None):
+
+    update_dict = {'status': status, 'billable_units': billable_units, 'updated_at': datetime.utcnow()}
+    if recipient_address:
+        update_dict['to'] = recipient_address
```

Comment:
For templated letters we only store the first line in `to` - is there a reason why we're now storing the whole address instead?

Reply:
We might want check if the address passed in are valid or search on that field. I think we would start to persist the whole address block on templated letters rather than restrict precompiled letters to the first line.